From 31bedf490c685a8ae3b3a589ecde3be40df62c01 Mon Sep 17 00:00:00 2001
From: Adam Szita <szita@cloudera.com>
Date: Fri, 12 May 2017 15:11:41 +0200
Subject: [PATCH 1108/1164] CDH-52465 Create and backport BeeLineDriver test
 changes to run qtests against a cluster

Updating expected test result for cascade_dbdrop_hadoop20

Change-Id: I8163ce9eaf3d6975f1067b7582a8700bdd36d716
---
 .../results/negative/cascade_dbdrop_hadoop20.q.out |   24 ++------------------
 1 file changed, 2 insertions(+), 22 deletions(-)

diff --git a/hbase-handler/src/test/results/negative/cascade_dbdrop_hadoop20.q.out b/hbase-handler/src/test/results/negative/cascade_dbdrop_hadoop20.q.out
index 4ff29e1..cef7a06 100644
--- a/hbase-handler/src/test/results/negative/cascade_dbdrop_hadoop20.q.out
+++ b/hbase-handler/src/test/results/negative/cascade_dbdrop_hadoop20.q.out
@@ -4,34 +4,14 @@ PREHOOK: Output: database:hbaseDB
 POSTHOOK: query: CREATE DATABASE hbaseDB
 POSTHOOK: type: CREATEDATABASE
 POSTHOOK: Output: database:hbaseDB
-PREHOOK: query: -- INCLUDE_HADOOP_MAJOR_VERSIONS(0.20, 0.20S)
--- Hadoop 0.23 changes the behavior FsShell on Exit Codes
--- In Hadoop 0.20
--- Exit Code == 0 on success
--- Exit code < 0 on any failure
--- In Hadoop 0.23
--- Exit Code == 0 on success
--- Exit Code < 0 on syntax/usage error
--- Exit Code > 0 operation failed
-
-CREATE TABLE hbaseDB.hbase_table_0(key int, value string)
+PREHOOK: query: CREATE TABLE hbaseDB.hbase_table_0(key int, value string)
 STORED BY 'org.apache.hadoop.hive.hbase.HBaseStorageHandler'
 WITH SERDEPROPERTIES ("hbase.columns.mapping" = ":key,cf:string")
 TBLPROPERTIES ("hbase.table.name" = "hbase_table_0")
 PREHOOK: type: CREATETABLE
 PREHOOK: Output: database:hbasedb
 PREHOOK: Output: hbaseDB@hbase_table_0
-POSTHOOK: query: -- INCLUDE_HADOOP_MAJOR_VERSIONS(0.20, 0.20S)
--- Hadoop 0.23 changes the behavior FsShell on Exit Codes
--- In Hadoop 0.20
--- Exit Code == 0 on success
--- Exit code < 0 on any failure
--- In Hadoop 0.23
--- Exit Code == 0 on success
--- Exit Code < 0 on syntax/usage error
--- Exit Code > 0 operation failed
-
-CREATE TABLE hbaseDB.hbase_table_0(key int, value string)
+POSTHOOK: query: CREATE TABLE hbaseDB.hbase_table_0(key int, value string)
 STORED BY 'org.apache.hadoop.hive.hbase.HBaseStorageHandler'
 WITH SERDEPROPERTIES ("hbase.columns.mapping" = ":key,cf:string")
 TBLPROPERTIES ("hbase.table.name" = "hbase_table_0")
-- 
1.7.9.5

